calour/__init__.py

```diff
@@ -8,3 +8,8 @@
 
 __credits__ = "https://github.com/biocore/calour/graphs/contributors"
 __version__ = "0.1.0.dev0"
+
+from calour.experiment import *
+from calour.io import *
+from calour.filtering import *
+from calour.sorting import *
```

Comment:
i think this is like in numpy
we import all functions from all modules so they are all available under calour.XXX
this is so the user does not need to remember in which module is function is located
so when you use calour, you can just do import calour and then use whatever function you want...